uebung3.py

```diff
@@ -0,0 +1,33 @@
+import math
+
+N = 3
+middle = (N*3)+2
+
+## Row 1
+print(" " * (middle-1), end="")
+print("y", end="")
+print(" " * (middle-1),end="\n\n")
+
+## Row 2
+print(" " * (middle-1), end="")
+print("^", end="")
+print(" " * (middle-1),end="\n")

```

Comment:
actually you're using this pattern `" " * (middle-1) + something + " " * (middle-1)` a lot, try out the following in a python shell:

```python
format_string = " " * (middle-1) + "{middle}" + " " * (middle-1)
format_string.format(middle='|')
```